app/controllers/referee_interface/reference_controller.rb

```diff
@@ -143,6 +142,16 @@ def finish
       @application_form = reference.application_form
     end
 
+    def decline
```

Comment:
I think this action should be called `confirm_decline` since it renders the confirmation page.

The one below should be `decline`